Library/Formula/pidcat.rb

```diff
@@ -8,6 +9,10 @@ class Pidcat < Formula
 
   def install
     bin.install "pidcat.py" => "pidcat"
+
+    if build.with? "bash-completion"
+      bash_completion.install "bash_completion.d/pidcat"
```

Comment:
Likewise, if it needs the dependency make this a single line `if` statement, not a block.